modules/codegen/src/main/scala/com/twilio/guardrail/SwaggerUtil.scala

```diff
@@ -321,146 +322,89 @@ object SwaggerUtil {
-        case x =>
-          for {
-            tpe <- fallbackPropertyTypeHandler(x)
-          } yield Resolved[L](tpe, None, None, None, None) // This may need to be rawType=string?
+      def buildResolve[B: Extractable, A <: Schema[_]: Default.GetDefault](transformLit: B => Free[F, L#Term]): Tracker[A] => Free[F, ResolvedType[L]] = { a =>
+        val rawType   = a.downField("type", _.getType())
+        val rawFormat = a.downField("format", _.getFormat())
+        for {
+          customTpeName <- customTypeName(a)
+          res <- (
+            typeName[L, F](rawType, rawFormat, customTpeName),
+            Default(a.get).extract[B].traverse(transformLit(_))
+          ).mapN(Resolved[L](_, None, _, rawType.get, rawFormat.get))
+        } yield res
       }
-      log.debug(s"property:\n${log.schemaToString(property)}") >> strategies(
+
+      log.debug(s"property:\n${log.schemaToString(property.get)}").flatMap { _ =>
         property
-      )
+          .refine[Free[F, ResolvedType[L]]]({ case a: ArraySchema => a })(
+            p =>
+              for {
+                items <- getItems(p.get)
+                rec   <- propMetaImpl[L, F](Tracker.hackyAdapt(items, Vector.empty))(strategy)
+                res <- rec match {
+                  case Resolved(inner, dep, default, _, _) =>
+                    (liftVectorType(inner), default.traverse(liftVectorTerm))
+                      .mapN(Resolved[L](_, dep, _, None, None): ResolvedType[L])
+                  case x: DeferredMap[L]   => embedArray(x)
+                  case x: DeferredArray[L] => embedArray(x)
+                  case x: Deferred[L]      => embedArray(x)
+                }
+              } yield res
+          )
+          .orRefine({ case m: MapSchema => m })(
+            m =>
+              for {
+                rec <- m
+                  .downField("additionalProperties", _.getAdditionalProperties)
+                  .get
+                  .fold[Free[F, ResolvedType[L]]](
+                    objectType(None).map(Resolved[L](_, None, None, None, None))
+                  )({
+                    case b: java.lang.Boolean =>
+                      objectType(None).map(Resolved[L](_, None, None, None, None))
+                    case s: Schema[_] => propMetaImpl[L, F](Tracker.hackyAdapt(s, Vector.empty))(strategy)
+                  })
+                res <- rec match {
+                  case Resolved(inner, dep, _, _, _) =>
+                    liftMapType(inner).map(Resolved[L](_, dep, None, None, None))
+                  case x: DeferredMap[L]   => embedMap(x)
+                  case x: DeferredArray[L] => embedMap(x)
+                  case x: Deferred[L]      => embedMap(x)
+                }
+              } yield res
+          )
+          .orRefine({ case ref: Schema[_] if Option(ref.get$ref).isDefined => ref })(ref => getSimpleRef(ref.get).map(Deferred[L]))
+          .orRefine({ case b: BooleanSchema => b })(buildResolve(litBoolean))
```

Comment:
Turns out these were all the same, with only two variants (has default vs no default possible)